lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -310,6 +314,38 @@ def create_test_xcconfig_files
             end
           end
 
+          # Creates a script that copies the resources to the bundle of the test target.
+          #
+          # @param [Symbol] test_type
+          #        The test type to create the script for.
+          #
+          # @return [void]
+          #
+          def create_test_target_copy_resources_script(test_type)
+            path = target.copy_resources_script_path_for_test_type(test_type)
+            pod_targets = [target, *target.test_dependent_targets]
+            resource_paths_by_config = { 'Debug' => pod_targets.flat_map(&:resource_paths) }
```

Comment:
For test targets we don't care for `Release` but I'd be happy to change it to loop through `build_configurations`